presto-druid/src/main/java/com/facebook/presto/druid/DruidPushdownUtils.java

```diff
@@ -42,6 +47,106 @@
 {
     private DruidPushdownUtils() {}
 
+    public enum ExpressionType
+    {
+        GROUP_BY,
+        AGGREGATE,
+    }
+
+    public static class GroupByColumnNode
```

Comment:
Having a public enum in a util class doesn't sound good. Can you make `AggregationColumnNode` a top-level class and call it `DruidAggregationColumnNode`? Put the enum and two implementations as the inner class of it.